### DISCUSSION TITLE
Welcome to Orchard Core Commerce Discussions!

### DISCUSSION THREAD

**Discussion by bleroy** [OrchardCMS/OrchardCore.Commerce#66] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect users, contributors, and maintainers of the Orchard Core Commerce project. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by agriffard** [comment]
Contributor on Orchard Core, I hope I will be able to bring some value in a few areas.

If you need some help of any kind, please don't hesitate to ask.

**Comment by bleroy** [comment]
I'm Bertrand Le Roy, the main maintainer of this module, and a contributor.

**Reply by sateeshs** [reply]
@bleroy I can help with dotnet core development tasks.

**Reply by bleroy** [reply]
Hi @sateeshs and welcome. Please take a look at the list of issues and let us know if there's something in there you'd like to work on.

**Comment by mwpowellhtx** [comment]
Chipping in here. Potentially contributing to _OC_. Definitely interested in _Commerce_ vis-a-vis _OC Cms_. Let me know what y'all do for meetings and such; at least voice, I find that far more effective to consider opportunities, challenges, etc.

**Reply by bleroy** [reply]
Welcome, Michael. the project is frankly currently not active enough to justify regular meetings. I'm the main contributor, on my free time, and I usually answer pretty fast to questions here.
